bundle/manifests/audit.policies.ibm.com_auditpolicies.yaml

```diff
@@ -15,50 +15,51 @@ spec:
-          properties:
-            clusterAuditPolicy:
-              description: enforce, inform
-              type: object
-            labelSelector:
-              description: selecting a list of namespaces where the policy applies
-              type: object
-            namespaceSelector:
-              description: namespaces on which to run the policy
-              properties:
-                exclude:
-                  items:
-                    type: string
-                  type: array
-                include:
-                  items:
-                    type: string
-                  type: array
-              type: object
-            remediationAction:
-              description: remediate or enforce
-              type: string
-          type: object
-        status:
-          properties:
-            auditDetails:
-              description: selecting a list of services to validate
-              type: object
-            compliant:
-              type: string
-          type: object
-  version: v1alpha1
+  versions:
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              clusterAuditPolicy:
+                description: enforce, inform
+                type: object
+              labelSelector:
+                description: selecting a list of namespaces where the policy applies
+                type: object
+              namespaceSelector:
+                description: namespaces on which to run the policy
+                properties:
+                  exclude:
+                    items:
+                      type: string
+                    type: array
+                  include:
+                    items:
+                      type: string
+                    type: array
+                type: object
+              remediationAction:
+                description: remediate or enforce
+                type: string
+            type: object
+          status:
+            properties:
+              auditDetails:
+                description: selecting a list of services to validate
+                type: object
+              compliant:
+                type: string
+            type: object
+  versions: v1alpha1
```

Comment:
You can remove this line...it is at line 18 now

Reply:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#customresourcedefinitionspec-v1-apiextensions-k8s-io

you are missing fields `name` `storage` `served` for the spec.versions array. Also, the spec.versions is an array, you need to reference the first object of the array as such. Refer to the MongoDB PR